### DISCUSSION TITLE
想要修改这个上位机的UI需要修改哪个文件中的代码呢？

### DISCUSSION THREAD

**Discussion by TTsXDU** [wh201906/SerialTest#49] [Q&A] [answered]
我想要对这个上位机的UI进行修改调整，请问UI设计对应的代码在哪个文件中呢？我是小白，恳请指教。

**Comment by wh201906** [comment, marked as answer]
你需要用Qt Creator修改项目当中的.ui文件

**Comment by TTsXDU** [comment]
好的，我去试试，感谢您的及时回复，谢谢。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "wh201906/SerialTest"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年2月19日(星期一) 上午9:45
***@***.***&gt;;
抄送:&nbsp;"Sire ***@***.******@***.***&gt;;
主题:&nbsp;Re: [wh201906/SerialTest] 想要修改这个上位机的UI需要修改哪个文件中的代码呢？ (Discussion #49)





 
你需要用Qt Creator修改项目当中的.ui文件
